action.yml

```diff
@@ -4,12 +4,17 @@ author: 'haya14busa'
 inputs:
   default_bump_level:
     description: "Default bump level if labels are not attached [major,minor,patch]. Do nothing if it's empty"
+    required: false
   dry_run:
     description: "Do not actually tag next version if it's true"
+    required: false
   github_token:
     description: 'GITHUB_TOKEN to list pull requests'
     default: '${{ github.token }}'
+    required: false
```

Comment:
This is actually required. It's just filled by default. Can you remove the required field or mark it as true?